Could not pass event PlayerInteractEvent to Khaos v1.0.4

```
[08:02:06] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to Khaos v1.0.4
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:320) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:529) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:514) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:441) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:408) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:404) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1293) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:28) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:1) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:19) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(SourceFile:144) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(SourceFile:118) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.MinecraftServer.aZ(MinecraftServer.java:917) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:910) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(SourceFile:127) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:894) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:827) [spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_232]
Caused by: java.lang.IllegalStateException: clickedBlock must not be null
        at blue.feelingso.khaos.KhaosListener.onPlayerRightClick(KhaosListener.kt:70) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor21.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_232]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_232]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316) ~[spigot-1.15.1.jar:git-Spigot-492a779-83af86d]
        ... 20 more
```

これSpigotが悪いのかな

--- a/build.gradle
+++ b/build.gradle
@@ -1,5 +1,5 @@
 group 'blue.feelingso'
-version '1.0.4-SNAPSHOT'
+version '1.0.5-SNAPSHOT'
 
 buildscript {
     ext.kotlin_version = '1.3.61'

--- a/src/main/kotlin/blue/feelingso/khaos/KhaosListener.kt
+++ b/src/main/kotlin/blue/feelingso/khaos/KhaosListener.kt
@@ -45,7 +45,7 @@ class KhaosListener(khaos :Khaos) : Listener {
         val conf = khaos.khaosConfig
         val player = ev.player
         val tool = player.inventory.itemInMainHand
-        val clickedBlock = ev.clickedBlock
+        val clickedBlock = ev.clickedBlock ?: return
 
         when(ev.action) {
             // allowToolsに記載されていないツールとブロックのセットなら無視する

--- a/src/main/resources/plugin.yml
+++ b/src/main/resources/plugin.yml
@@ -1,5 +1,5 @@
 name: Khaos
-version: 1.0.4
+version: 1.0.5
 main: blue.feelingso.khaos.Khaos
 api-version: "1.15"
 commands:

